.github/workflows/publish.yaml

```diff
@@ -0,0 +1,58 @@
+name: Image Checks
+
+on:
+  push:
+    # Run only on release pushes
+    tags:
+      - v*
```

Comment:
Could: It might be an idea to keep the same release conventions across our repos. For python projects we just have a semver tag (no "v" prefix, except for ones in the bluesky org where we do). I'm not hugely fussed about keeping the "v" but I think it should be documented.

Reply:
I've changed it to use the [type=semver](https://github.com/docker/metadata-action?tab=readme-ov-file#typesemver) tagging from the metadata action which expects the tag to be `vX.Y.Z` but creates the image tag as `X.Y.Z`. The alternative `pep440` sounds like it was made to deal with python conventions.